app/modules/rfswitch.c

```diff
@@ -51,14 +53,34 @@ typedef struct Protocol {
  * These are combined to form Tri-State bits when sending or receiving codes.
  */
 static const Protocol proto[] = {
-  { 350, {  1, 31 }, {  1,  3 }, {  3,  1 }, false },    // protocol 1
-  { 650, {  1, 10 }, {  1,  2 }, {  2,  1 }, false },    // protocol 2
-  { 100, { 30, 71 }, {  4, 11 }, {  9,  6 }, false },    // protocol 3
-  { 380, {  1,  6 }, {  1,  3 }, {  3,  1 }, false },    // protocol 4
-  { 500, {  6, 14 }, {  1,  2 }, {  2,  1 }, false },    // protocol 5
-  { 450, { 23,  1 }, {  1,  2 }, {  2,  1 }, true }      // protocol 6 (HT6P20B)
+  { 350, {  1, 31 }, {  1,  3 }, {  3,  1 }, false, 0, false },    // protocol 1
+  { 650, {  1, 10 }, {  1,  2 }, {  2,  1 }, false, 0, false },    // protocol 2
+  { 100, { 30, 71 }, {  4, 11 }, {  9,  6 }, false, 0, false },    // protocol 3
+  { 380, {  1,  6 }, {  1,  3 }, {  3,  1 }, false, 0, false },    // protocol 4
+  { 500, {  6, 14 }, {  1,  2 }, {  2,  1 }, false, 0, false },    // protocol 5
+  { 450, { 23,  1 }, {  1,  2 }, {  2,  1 }, true,  0, false }     // protocol 6 (HT6P20B)
 };
 
+Protocol custom = { 350, {  1, 31 }, {  1,  3 }, {  3,  1 }, false, false };  // Defaults to same as protocol 1
```

Comment:
It would be nice if this was the same as protocol 1 -- there is a missing 0 near the end of the initialization. It doesn't really make a difference, but it ought to be fixed.